src/main/resources/CodeBuilder/config.jelly

```diff
@@ -97,6 +97,14 @@
           <f:textbox />
         </f:entry>
 
+        <f:entry title="Inclusion pattern (optional)" field="workspaceIncludes" help="/plugin/aws-codebuild/help-workspaceIncludes.html">
+          <f:textbox />
+        </f:entry>
+
+        <f:entry title="Exclusion pattern (optional)" field="workspaceExcludes" help="/plugin/aws-codebuild/help-workspaceExcludes.html">
+          <f:textbox />
+        </f:entry>
+
```

Comment:
This will look like this:
![image](https://user-images.githubusercontent.com/3115961/155284002-180acd2a-1183-47fe-91d9-7ea45e5a2ea2.png)